Secure machine fixes + customer feedback (#443)
foundriesio/docs: source/reference-manual/security/secure-boot-imx-ahab.rst

.. _ref-secure-boot-imx-ahab:

> Secure Boot on i.MX 8/8X Families using AHAB

Reviewer: It is not clear here or in the NXP documentation that 8QM is in this category.  You have to dig into documentation in NXP to see that this includes 8QM. We might want to just add `including 8QM` in the title or first line.

The Security Reference Manual for your specific SoC indicates which fuses need to be programed with the SRK fuse information.

i.MX 8QM fusing
--------------------------

> On the i.MX 8QM SoC fuses are stored in fuse bank 0, words 722 to 737::

Reviewer: We need to make it clear that these are example values and not meant for production.

        => fuse prog -y 0 736 0x6E0B791C
        => fuse prog -y 0 737 0x6A558134

> Alternatively, you can program SoC fuses via SDP using NXP's Universal Update Utility with a script as follows::

Reviewer 2: ```suggestion
Alternatively, you can program SoC fuses via SDP  by using NXP's Universal Update Utility. This is shown in the following script::
```
